src/main/java/com/futurcollars/lesson10/zad2/WriteToTheSecondLine.java

```diff
@@ -0,0 +1,23 @@
+package com.futurcollars.lesson10.zad2;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class WriteToTheSecondLine {
+
+    public void getWriteTheSecondLine() {
```

Comment:
tutaj tak samo nazwa do poprawy, nie używaj dwóch czasowników obok siebie, niech to ma sens składniowy. Czyli albo get albo write albo read, a nie wszystkie naraz :)